Riot/Modules/directory/people/detail/cells/PeopleProfileTableViewCell.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  PeopleProfileTableViewCell.swift
+//  Riot
+//
+//  Created by Naurin Afrin on 5/8/20.
+//  Copyright © 2020 matrix.org. All rights reserved.
+//
+
+import UIKit
+
+class PeopleProfileTableViewCell: UITableViewCell {
+    
+    var person: ActPeople!
+    @IBOutlet weak var avatar: MXKImageView!
+    @IBOutlet weak var name: UILabel!
+    @IBOutlet weak var jobTitle: UILabel!
+    @IBOutlet weak var organisation: UILabel!
+    @IBOutlet weak var businessUnit: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+        
+    }
+    @IBAction private func StartChatOption(_ sender: Any) {
+        AppDelegate.theDelegate().startDirectChat(withUserId: person.baseUser.userId, completion: {
+            
+        })
+    }
+    @IBAction private func StartAudioCallOption(_ sender: Any) {
+        placeCallWithUser(withVideo: false)
+    }
+    @IBAction private func StartVideoCallOption(_ sender: Any) {
+        placeCallWithUser(withVideo: true)
+    }
+    
+    func placeCallWithUser(withVideo video: Bool) {
+        guard let userID = person.baseUser.userId else { return }
+        if let theSession = (AppDelegate.theDelegate().mxSessions.first as? MXSession) {
+            if let room: MXRoom = theSession.directJoinedRoom(withUserId: userID) {
+                room.placeCall(withVideo: video, completion: {(callResult) in
+                    if callResult.isFailure {
+                        
```

Comment:
Please add a TODO comment that this should be replaced once the back end is available

Reply:
Added TODO message at this point.